packages/sitecore-jss/src/i18n/dictionary-service.ts

```diff
@@ -0,0 +1,63 @@
+import mcache from 'memory-cache';
+
+const defaults = {
+  cacheTimeout: 60,
+};
+
+/**
+ * Phrases from the Sitecore Dictionary Service
+ */
+export interface DictionaryPhrases {
+  [k: string]: string;
+}
+
+/**
+ * Cache options for dictionary data services
+ */
+export interface CacheOptions {
```

Comment:
I think the generic name is fine, since this interface could be used for other things in theory. In anything, I think I should remove the mention of dictionary service from the comment.

Reply:
It's fine since we don't export this publicly.